deploy/crds/kieapp.crd.yaml

```diff
@@ -605,3 +605,9 @@ spec:
                     replaceRole:
                       type: boolean
                       description: Whether to add to the current roles, or replace the current roles with the mapped ones. Replaces if set to true.
+            upgradeProductPatch:
+              type: boolean
+              description: Set false to disable automatic product patch version upgrades, it is enabled by default.
+            upgradeProductMinor:
+              type: boolean
+              description: Set true to enable automatic product minor version upgrades, it is disabled by default.
```

Comment:
Let's do a struct called Upgrades, and then just minor & patch in there.

Reply:
done